clapper/main.ml

```diff
@@ -0,0 +1,11 @@
+open Printf
+open Clapper
+
+let main () =
+  (* How do I do compound lets? *)
```

Comment:
Couldn’t figure this out; do I have to repeat `let … in` when I’m in a compound statement?

Reply:
Nah, you just separate them with `and` I'm pretty sure. `let x = 5 and y = 5 in x + y`

Reply:
Oh interesting, `and` means mutual recursion. I thought `let rec` did that but that must do self-recursion explicitly. Neat that you can control all that I guess. https://codereview.stackexchange.com/questions/15558/is-this-use-of-let-and-in-ocaml-poor-style